cfpb/cfgov-refresh
# Remove classname='full' from Wagtail editor fields

Theoretically, passing the `classname='full'` argument to a Wagtail field panel [will make the field render at something closer to the full width of the page than the 
default](https://docs.wagtail.io/en/v2.3/reference/pages/panels.html#full-width-input). 
In practice, in the places we were using them, it was doing almost nothing in Wagtail 1, and is misaligning things (`BooleanFields` and `RichTextFields`, in particular) in Wagtail 2.

This PR removes all instances of that argument.

## Removals

- `classname='full'` removed from all Wagtail field panels

## Testing

1. Pull branch
1. Open a job listing page for editing and confirm that the checkbox and rich text fields appear as in the Expected screenshot on https://GHE/CFGOV/platform/issues/3694

## Screenshots

### Before

![new & busted](https://user-images.githubusercontent.com/1044670/76033516-6b232b00-5f0a-11ea-9d83-48a61b08dbfe.png)

![old hotness](https://user-images.githubusercontent.com/1044670/76033547-7c6c3780-5f0a-11ea-948d-c163e2a68aa6.png)

## Checklist


## Testing checklist

### Browsers
